tests/integration/courses.test.js

```diff
@@ -3354,48 +3399,49 @@ describe('COURSES ROUTES - DELETE /courses/{_id}/trainee/{traineeId}', () => {
       });
     });
 
-    it('should return 200 if holding admin delete learner from his holding', async () => {
-      authToken = await getTokenByCredentials(holdingAdminFromOtherCompany.local);
-      const res = await app.inject({
+    it('should return 403 as user is trainer if not one of his courses', async () => {
```

Comment:
Que penses-tu de : `should return 403 because the user is a trainer but this is not one of his courses`

Reply:
done